cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -285,43 +258,37 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArrayPlex : public WrapBase<Wr
                     return execute_callback_with_typed_array(
                         self, [](const auto & array)
                         {
-                            pybind11::tuple ret(array.shape().size());
-                            for (size_t i = 0; i < array.shape().size(); ++i)
-                            {
-                                ret[i] = array.shape()[i];
-                            }
-                            return ret; });
+                            using data_type = typename std::remove_const_t<std::remove_reference_t<decltype(array[0])>>;
```

Comment:
This is too magical.  The original lambda is much more readable.  Please don't use this overly tricky template code.

Reply:
I see. I thought it is readable if both wrapper class call the same function.
So, do you want to revert the change for both "shape" and "stride"?

Reply:
I think the original lambda is good for this occurrence.  Specifically here reverting is good.

In other occurrences maybe more review is needed.  I am fine to use this PR or another one to handle the cases that need review.  A decision should be made quickly.

Reply:
reverted.